examples/bri-3/src/bri/identity/bpiAccounts/agents/bpiAccountsStorage.agent.ts

```diff
@@ -63,8 +63,8 @@ export class BpiAccountStorageAgent extends PrismaService {
         bp.ownerBpiSubjectAccounts.map((o) => {
```

Comment:
now that the prisma model and domain model are in sync, we can directly invoke mapper(o, BpiSubjectAccount,BpiSubjectAccount)